### PR TITLE
(peazip) migrated from dtgm

### PR DESCRIPTION
**Pull request by dimqua**
## Description
Embedded and updated, although it's not in perfect state yet. :-)

## Motivation and Context
Package is outdated and unmaintained.

## How Has this Been Tested?
testing environment
## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Migrated package (a package have been migrated from another repository)

## Checklist:
- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this mean usually the notes in the description of a package).
- [x] I have updated the documentation accordingly (this mean usually the notes in the description of a package).
- [x] All files is up to date with the latest [Contributing Guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [x] The added/modified package passed install/uninstall in the chocolatey test environment.
- [ ] The changes only affect a single package (not including meta package).

## Original Location
- [Original Repository](https://github.com/dtgm/chocolatey-packages/)

* [ ] TBD. *Include the link to the opened PR that removes the package from the original location*
* [ ] The [migration guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Package-migration-process) have been followed

### CODE REVIEW

File: automatic/peazip/peazip.nuspec [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+The program supports over 150 archive formats including 7Z, ACE, ARC, ARJ, BZ2, CAB, DMG, GZ, ISO, LHA, PAQ, PEA, RAR, TAR, UDF, WIM, ZIP, ZIPX...
+
+Features of PeaZip includes extract, create and convert multiple archives at once, create self-extracting archives, split/join, encrypted password manager, strong encryption with two factor authentication, secure deletion, find duplicate files, calculate hashes, manage graphic files (rotate, crop, resize, convert).
+    </description>
+    <projectUrl>http://www.peazip.org/</projectUrl>
+    <projectSourceUrl>https://osdn.net/projects/peazip/releases/</projectSourceUrl>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/peazip</packageSourceUrl>
+    <docsUrl>http://www.peazip.org/peazip-help.html</docsUrl>
+    <bugTrackerUrl>https://sourceforge.net/p/peazip/tickets/</bugTrackerUrl>
+    <tags>file-manager file-encryption file-compression zip rar 7zip tar admin peazip</tags>
```
Comment (majkinetor) [inline automatic/peazip/peazip.nuspec]: missing mandatory tag: `foss`

File: automatic/peazip/peazip.nuspec [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <licenseUrl>http://www.peazip.org/peazip-tos-privacy.html</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <copyright>Copyright © PeaZip srl</copyright>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/b9b0ed4a/icons/peazip.svg</iconUrl>
+    <releaseNotes>[PeaZip changelog](http://www.peazip.org/changelog.html)</releaseNotes>
+  </metadata>
+  <files>
+    <file src="tools\**" target="tools" />
```
Comment (majkinetor) [inline automatic/peazip/peazip.nuspec]: missing `legal` folder

File: automatic/peazip/tools/chocolateyinstall.ps1 [hunk 3]
```diff
@@ -0,0 +1,23 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+$fileName = if ((Get-ProcessorBits 64) -and ($env:chocolateyForceX86 -ne 'true')) {
```
Comment (majkinetor) [inline automatic/peazip/tools/chocolateyinstall.ps1]: checking bitness is no longer necessary, Depend on latest choco.

File: automatic/peazip/tools/chocolateyinstall.ps1 [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  softwareName   = 'PeaZip'
+  file           = "$toolsDir\$fileName"
+  silentArgs     = '/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP-'
+  validExitCodes = @(0, 1223)
+}
+
+Install-ChocolateyInstallPackage @packageArgs
+
+# Lets remove the installer as there is no more need for it
+Remove-Item -Force "$toolsDir\*.exe","$toolsDir\*.exe.ignore" -ea 0
```
Comment (majkinetor) [inline automatic/peazip/tools/chocolateyinstall.ps1]: Exe files must be removed with 

```

ls $toolsDir\*.exe | % { rm $_ -ea 0; if (Test-Path $_) { sc "$_.ignore" } }
```

File: automatic/peazip/tools/chocolateyuninstall.ps1 [hunk 5]
```diff
@@ -0,0 +1,16 @@
+﻿$packageName = 'peazip'
```
Comment (majkinetor) [inline automatic/peazip/tools/chocolateyuninstall.ps1]: Should be modernized

File: automatic/peazip/tools/chocolateyinstall.ps1 (hunk 4 above)
Comment (pauby) [reply automatic/peazip/tools/chocolateyinstall.ps1]: Don't use aliases in the scripts. Use the full cmdlet name so `Get-ChildItem` instread of `ls` for example.

File: automatic/peazip/tools/chocolateyinstall.ps1 [hunk 6]
```diff
@@ -2,22 +2,16 @@
...
-  file           = "$toolsDir\$fileName"
   silentArgs     = '/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP-'
   validExitCodes = @(0, 1223)
 }
 
 Install-ChocolateyInstallPackage @packageArgs
 
-# Lets remove the installer as there is no more need for it
-Remove-Item -Force "$toolsDir\*.exe","$toolsDir\*.exe.ignore" -ea 0
+ls $toolsDir\*.exe | % { rm $_ -ea 0; if (Test-Path $_) { sc "$_.ignore" } }
```
Comment (pauby) [inline automatic/peazip/tools/chocolateyinstall.ps1]: Aliases shouldn't be used in scripts.

Comment (majkinetor) [reply automatic/peazip/tools/chocolateyinstall.ps1]: [1.1.7](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md#117-clean-code) allows standard aliases.

Comment (pauby) [reply automatic/peazip/tools/chocolateyinstall.ps1]: *Use aliases sparingly. **If you use them, limit their use to the default aliases which are read-only (can not be modified by the user)**, well known, and will be supported in the future versions of Powershell. Also ensure they are compatible with PowerShell v2+.* (my emphasis)

1,1.7 allows aliases that are read-only. `ls`, `rm` and `sc` are not read-only. `%` and `?` are.

Comment (majkinetor) [reply automatic/peazip/tools/chocolateyinstall.ps1]: That is then error in guidelines.

**Comment by dimqua** [comment]
Thx. I will create a new PR then.

**Comment by AdmiringWorm** [comment]
@dimqua you don't need to create a new PR, just reuse this one

**Pull request closed without merging**